cmritPath.py

```diff
@@ -32,16 +32,20 @@ def __init__(self):
         # Basic Science Node
         # we can add nodes & edges like this..
         G.add_node('basic_science', y=12.96626, x=77.71211, street_count=3)
-        G.add_edge('teacher_parking', 'basic_science', length=200)
+        # G.add_edge('teacher_parking', 'basic_science', length=200)
+        G.add_edges_from([('teacher_parking','basic_science'),('basic_science','teacher_parking')],length=300) # Used to add multiple edges at once
 
         # Ganesh statue node
         G.add_node('ganesha_statue', y=12.96598, x=77.71148, street_count=3)
-        G.add_edge('basic_science', 'ganesha_statue', length=200)
+        # G.add_edge('basic_science', 'ganesha_statue', length=200)
+        G.add_edges_from([('basic_science','ganesha_statue'),('ganesha_statue','basic_science')],length=300)
```

Comment:
Why the modification from add_edge() to add_edges_from()?

Reply:
I don't think there is any computational changes as far as I know this just increases code readability and decreases the length of the code

Reply:
G.add_edge('basic_science', 'ganesha_statue', length=300) 

 G.add_edges_from([('basic_science','ganesha_statue'),('ganesha_statue','basic_science')],length=300)

I don't feel the need of using this method. **add_edges_from**() looks like it has some other specific functionality.
No gain any way. Let us use the right method of approach.

Reply:
it is used for adding multiple edges together. here u rnt
modify and check  if all nodes can be linked in one go with **add_edges_from()**

Reply:
ok